runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -95,9 +95,12 @@ function TSHighlighter.new(tree, opts)
   -- syntax FileType autocmds. Later on we should integrate with the
   -- `:syntax` and `set syntax=...` machinery properly.
   if vim.g.syntax_on ~= 1 then
-    vim.api.nvim_command('runtime! syntax/synload.vim')
+    vim.cmd.runtime({ 'syntax/synload.vim', bang = true })
```

Comment:
```suggestion
    vim.cmd.runtime { 'syntax/synload.vim', bang = true }
```

Reply:
I would prefer that too, but it goes against our linter rules 😢